lib/mavenlink/integrated_resource.rb

```diff
@@ -0,0 +1,24 @@
+module Mavenlink
+  class IntegratedResource < Model
+    def save
+      if valid?
+        reload(create_or_update(attributes_for_create_or_update))
```

Comment:
@shirish-pampoorickal I think we are reloading the attributes that comes back from big maven into the konnekti Mavenlink Model. There shouldn't be any changes, but since the other models are doing this too, it might be safer to keep the attributes up to date.

A + G